src/components/SectionWarranty/styles.js

```diff
@@ -0,0 +1,32 @@
+import styled from "@emotion/styled";
+
+const style = (Component) => styled(Component)`
+  .titulo-strong {
+    font-family: Lato;
```

Comment:
If Lato is the chosen font for every text, apply it globally instead of repeating it for each class individually.

Reply:
Made Lato global font-style in commit 898e81c.